jobs/sartregroup_NY_pythondatadev.html

```diff
@@ -0,0 +1,58 @@
+---
+title: Python Developer - HFT
+company: Sartre Group (Agency) on behalf of GTS.
+location: New York, US
+contract: Permanent
+contact:
+    name: Joe Jolly
+    email: joe.jolly@sartregroup.com
+    phone: +1 917 795 0584
+created: !!timestamp '2017-01-25'
+tags:
+  - python
+  - expert
+  - scientific libraries
```

Comment:
You can mention specific scientific libraries here.  I don't think people would ever use this search term.